venues/OpenReview.net/Support/init.py

```diff
@@ -339,87 +339,222 @@
         'signatures': {
             'values-regex': '~.*'
         },
-        'content': reviewer_recruitment_content
+        'content': recruitment_content
     }
 }))
 
-ac_recruitment_content = {
-    'title': {
-        'value': 'Area chair recruitment',
-        'required': True,
-        'order': 1
+bid_stage_content = {
+    'bid_start_date': {
+        'description': 'When does bidding on submissions begin? Please use the format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59)',
```

Comment:
We need timezone information. Or tell them it is in UTC.

Reply:
Agreed. This is missing from the request form also. We discussed a few ideas regarding this. Placing some text about this in the description was insufficient as once a reply note is submitted, the note does not render the field description. Let me know if you have any ideas.